eng/pipelines/bicep-warnings.yml

```diff
@@ -51,3 +51,7 @@ stages:
               containerName: 'azure-sdk-for-js'
               blobPrefix: 'bicep-lint-warnings/$(DATE)'
               AdditionalArgumentsForBlobCopy: '--overwrite=true'
+    variables:
```

Comment:
I would move this to the top for clarity/convention

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/31353/commits/98b250e97766eda0c3d2fdf37b984777d0c4b18d